packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -52,15 +53,9 @@ function Spec(attrs) {
     function() {
       return '';
     };
-  this.expectationResultFactory = attrs.expectationResultFactory ||
-    function() {};
   this.queueRunnerFactory = attrs.queueRunnerFactory || function() {};
   this.throwOnExpectationFailure = !!attrs.throwOnExpectationFailure;
 
-  if (!this.queueableFn.fn) {
-    this.pend();
```

Comment:
If I understand correctly `fn` can be undefined when test function is not passed:
```js
test('test name'); // no test callback
```
But it looks like behaviour didn't change (Jest still marks such tests as skipped). So I think we're OK with removing this. cc: @dmitriiabramov